### ISSUE TITLE
Multiple parameter custom activation function

### ISSUE THREAD

**Issue by tawe141** [keras-team/keras#7658]
How should I implement an activation such as exp(A+B/x), where A and B are unknown constants to be optimized and x is the input variable?

**Comment by mahnerak** [comment]
Just create new custom Keras Layer (https://keras.io/layers/writing-your-own-keras-layers/),
then, instead of defining a layer with activation (say `Dense(units=40, activation='softmax')`), use your layer:
```python
tmp = Dense(units=40) (x)
y = MyCustomActivationLayer() (tmp)
```
